perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ModulesUtilsBlImpl.java

```diff
@@ -1183,6 +1184,18 @@ public User getUserFromMessage(PerunSessionImpl sess, String message) {
 		return convertedRanges;
 	}
 
+	@Override
+	public User getUserByLoginInNamespace(PerunSession sess, String login, String namespace) {
+		String attrName = AttributesManager.NS_USER_ATTR_DEF + ":" + AttributesManager.LOGIN_NAMESPACE + ":" + namespace;
+		List<User> usersByLoginInNamespace = getPerunBl().getUsersManagerBl().getUsersByAttributeValue(sess, attrName, login);
+
+		if (usersByLoginInNamespace.isEmpty()) {
+			return null;
+		} else if (usersByLoginInNamespace.size() == 1) {
+			return usersByLoginInNamespace.get(0);
+		} else throw new ConsistencyErrorException("There are 2 same logins '" + login + "' in namespace " + namespace + ".");
```

Comment:
It is 2 or more (size could be bigger than 2).

Reply:
Fixed